src/test/java/redis/clients/jedis/util/JsonParserTestUtil.java

```diff
@@ -0,0 +1,81 @@
+package redis.clients.jedis.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.google.gson.*;
+import redis.clients.jedis.json.parser.JsonParser;
+
+import java.lang.reflect.Type;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalAccessor;
+
+public class JsonParserTestUtil {
+  public static CustomJacksonParser getCustomJacksonParser() {
+    ObjectMapper om = new ObjectMapper();
+    om.registerModule(new JavaTimeModule());
+    om.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    return new CustomJacksonParser(om);
+  }
+  public static class CustomJacksonParser implements JsonParser {
+    private final ObjectMapper om;
+    CustomJacksonParser(ObjectMapper om) {
+      this.om = om;
+    }
+    @Override
+    public <T> T fromJson(String value, Class<T> valueType) {
+      try {
+        return om.readValue(value, valueType);
+      } catch (JsonProcessingException e) {
+        e.printStackTrace();
+      }
+      return null;
```

Comment:
Don't just print. Perhaps:
```suggestion
      try {
        return om.readValue(value, valueType);
      } catch (JsonProcessingException e) {
        throw new JedisException(e);
      }
```